frontends/web/src/components/amount/amount.test.tsx

```diff
@@ -108,13 +108,21 @@ describe('Amount formatting', () => {
         const { container } = render(<Amount amount="42" unit={coin} removeBtcTrailingZeroes/>);
         expect(container).toHaveTextContent('42');
       });
-      it('10.00000000 ' + coin + ' with removeBtcTrailingZeroes disabled stays 10.00000000', () => {
-        const { container } = render(<Amount amount="10.00000000" unit={coin}/>);
-        expect(container).toHaveTextContent('10.00000000');
+      it('10.00000000 ' + coin + ' with removeBtcTrailingZeroes disabled gets spaced', () => {
+        const { getByTestId } = render(<Amount amount="10.00000000" unit={coin}/>);
+        const blocks = getByTestId('amountBlocks');
+        expect(blocks.innerHTML).toBe(
+          '<span>10.00</span>' +
+          '<span class="space">000</span>' +
+          '<span class="space">000</span>');
       });
-      it('10.12300000 ' + coin + ' with removeBtcTrailingZeroes disabled stays 10.12300000', () => {
-        const { container } = render(<Amount amount="10.12300000" unit={coin}/>);
-        expect(container).toHaveTextContent('10.12300000');
+      it('10.12300000 ' + coin + ' with removeBtcTrailingZeroes disabled gets spaced', () => {
```

Comment:
pls add a case or modify a case where the part before the decimal point is large, e.g. 12345.67800000`

maybe also one where all decimals are used: `10.12345678`